tests/marray/marray_operators.h

```diff
@@ -822,13 +822,22 @@ class check_marray_pre_unary_operators_for_type {
     INFO("prefix unary operators for type \"" << type_name << "\": ");
 
     const auto num_elements = marray_common::get_num_elements();
-
-    static const auto unary_operators =
-        named_type_pack<op_upos, op_uneg, op_pre_inc, op_pre_dec, op_lnot,
-                        op_bnot>::generate("unary +", "unary -", "pre ++",
-                                           "pre --", "!", "~");
+    static const auto unary_operators = make_unary_operators();
```

Comment:
Isn't it a good opportunity for a lambda with immediate invocation?

Reply:
It is!